packages/mui-material/src/Select/Select.js

```diff
@@ -16,11 +17,18 @@ import useThemeProps from '../styles/useThemeProps';
 import useForkRef from '../utils/useForkRef';
 import { styled } from '../zero-styled';
 import rootShouldForwardProp from '../styles/rootShouldForwardProp';
+import { getSelectUtilityClasses } from './selectClasses';
 
 const useUtilityClasses = (ownerState) => {
   const { classes } = ownerState;
 
-  return classes;
+  const slots = {
+    root: ['root'],
+  };
+
+  const composedClasses = composeClasses(slots, getSelectUtilityClasses, classes);
+
+  return { ...classes, ...composedClasses };
```

Comment:
This implementation is taken from here
https://github.com/mui/material-ui/blob/75c24b0828981651082a3eede027b6871c96c440/packages/mui-material/src/Button/Button.js#L39-L42.

without spreaing `.MuiTablePagination-select` isn't getting applied correctly. https://mui.com/material-ui/api/table-pagination/#table-pagination-classes-select